node_modules/gh-core/lib/api.js

```diff
@@ -74,6 +74,9 @@ var init = module.exports.init = function(config, callback) {
                         return callback(err);
                     }
 
+                    // Initialise the Pattern listener
+                    require('gh-series/lib/internal/patterns/cambridge/listener');
+
```

Comment:
If we ever get to a point where we need different patterns for different institutions, we'll probably need some sort of registration mechanism. However, I think explicitly defining it is fine for now.